tests/test_deploy_egg.py

```diff
@@ -35,7 +35,8 @@ def tearDown(self):
     def test_parses_project_information_correctly(self):
         # this test's assertions are based on the values
         # defined on this folder's setup.py file
-        shutil.copytree('tests/samples/deploy_egg_sample_project/', self.tmp_dir)
+        shutil.rmtree(self.tmp_dir)
+        shutil.copytree('tests/samples/deploy_egg_sample_project', self.tmp_dir)
```

Comment:
Had started failing due to `self.tmp_dir` existing already, not sure how it worked before either.